src/components/_modal/pages/branchLocalizations/BranchLocalizations.js

```diff
@@ -18,15 +18,15 @@ import { LinkForBranch } from "@/components/_modal/pages/base/localizationsBase/
  * @param {Array.<string>} pids
  * @param {Object} agency
  * @param {boolean} onlyHoldingsOnAgency
+ * @param localizationOwnsPids
  * @returns {JSX.Element}
  */
 function SpecificInformationOnAgency({ pids, agency, onlyHoldingsOnAgency }) {
+  const availabilityAccumulated = agency?.availabilityOnAgencyAccumulated;
+
   return (
     <div className={styles.agency_holdings_row_wrapper}>
-      <AvailabilityLight
-        availabilityAccumulated={agency.availabilityOnAgencyAccumulated}
-        pickupAllowed={agency?.pickupAllowed}
-      />
+      <AvailabilityLight availabilityAccumulated={availabilityAccumulated} />
       <div className={styles.agency_holdings_result}>
         <Text type="text2">
           {Translate({
```

Comment:
den er en ret stor en, her kunne man godt trække oversætningen ud i en variabel for sig og så bruge variablen er nede.

Reply:
Jeps